packages/web/src/scss/components/CheckboxField/README.md

```diff
@@ -45,7 +45,19 @@ Hidden label:
 </label>
 ```
 
-Error state:
+Danger state:
```

Comment:
Please mention the `docs/DICTIONARIES.md` here as well.

Reply:
done